gravitee-node-license/src/main/java/io/gravitee/node/license/DefaultLicenseFactory.java

```diff
@@ -47,8 +47,11 @@ public DefaultLicenseFactory(LicenseModelService licenseModelService) {
     }
 
     @Override
-    public License create(@Nonnull String referenceType, @Nonnull String referenceId, @Nonnull String base64License)
+    public License create(@Nonnull String referenceType, @Nonnull String referenceId, String base64License)
         throws InvalidLicenseException, MalformedLicenseException {
+        if (Objects.isNull(base64License)) {
+            return new OSSLicense(referenceType, referenceId);
+        }
```

Comment:
Could be moved into the other create method taking byte array no? That way we cover both the case (nul String or byte[]). WDYT?

public License create(@Nonnull String referenceType, @Nonnull String referenceId, byte[] bytesLicense)

Reply:
Yes I can do that 👍